packages/blockchain/src/index.ts

```diff
@@ -115,7 +143,8 @@ export default class Blockchain implements BlockchainInterface {
   db: LevelUp
   dbManager: DBManager
 
-  private _genesis?: Buffer // the genesis hash of this blockchain
+  private _genesisBlock?: Block /** The genesis block of this blockchain */
```

Comment:
This is actually also a nice replacement to have the `Block` here and not level down to just the hash (why?).